src/core/Form/TextInput/TextInput.md

```diff
@@ -60,3 +65,20 @@ const status = errorState ? 'error' : 'default';
   </Button>
 </>;
 ```
+
+```js
+import { TextInput } from 'suomifi-ui-components';
+
+<>
+  <TextInput
+    labelText="TextInput with numbers"
+    type="number"
+    value={123}
```

Comment:
I'd use defaultValue for these to allow playing around with the examples. With a set value editing is not possible.

Reply:
Example is now using `defaultValue` instead of the previous `value`